docs/guides/getting-started/quick-start/_commands.mdx

```diff
@@ -0,0 +1,44 @@
+Tauri lets you enhance your Frontend with native capabilities. We call these [Commands][command], essentially Rust functions that you can call from your frontend JavaScript. This enables you to handle heavy processing or calls to the OS in much more performant Rust code.
+
+Let's make a simple example:
+
+```rust
+#[tauri::command]
+fn greet(name: &str) -> String {
+   format!("Hello, {}!", name)
+}
+```
+
+A Command is just like any regular Rust function, with the addition of the `#[tauri::command]` attribute macro that allows your function to communicate with the JavaScript context.
+
+Lastly, we also need to tell Tauri about our newly created command so that it can route calls accordingly. This is done with the combination of the `.invoke_handler()` function and the `generate_handler![]` macro you can see below:
+
+```rust
+fn main() {
+  tauri::Builder::default()
+  // highlight-next-line
+    .invoke_handler(tauri::generate_handler![greet])
+    .run(tauri::generate_context!())
+    .expect("error while running tauri application");
+}
+```
+
+Now we're ready to call your Command from the JavaScript frontend! We would normally be recommending the `@tauri-apps/api` package here, but since we're not using a bundler for this guide, please enable [`withGlobalTauri`][withglobaltauri] in your `tauri.conf.json` file.
+
+```json
+{
+  "build": {
+    "beforeBuildCommand": "",
+    "beforeDevCommand": "",
+    "devPath": "..",
+    "distDir": "..",
+    // highlight-next-line
+    "withGlobalTauri": true
+  },
+```
+
+This will inject a pre-bundled version of the API functions into your Frontend.
+
+:::tip
+If you want to know more about the communication between Rust and JavaScript, please read the Tauri [Inter-Process Communication][inter-process-communication] guide.
+:::
```

Comment:
This is included in the vite guide but probably needs to only be mentioned in vanilla guide. vite and other bundlers should use the npm package.

Reply:
Could add a bool in the fragment to conditionally render this bit

Reply:
Latest commit should fix this @amrbashir

Reply:
nice